cartographer_ros/configuration_files/assets_writer.lua

```diff
@@ -7,11 +7,11 @@ options = {
       -- action = "fixed_ratio_sampler",
       -- sampling_ratio = 1.,
     -- },
-    -- {
-      -- action = "min_max_range_filter",
-      -- min_range = 1.,
-      -- max_range = 150.,
-    -- },
+    {
```

Comment:
Should this be enabled by default? Also, laser fans contain their min/max range. Maybe that should be respected?